tutorials/nbconverted/single_cell_usage.py

```diff
@@ -0,0 +1,227 @@
+# 
+# From the mentioned repo, we specifically used this [dataset](https://github.com/WayScience/Benchmarking_NF1_data/tree/main/4_processing_features/data/Plate2/CellProfiler) and the associated [metadata](https://github.com/WayScience/Benchmarking_NF1_data/tree/main/3_extracting_features/metadata) to generate the walkthrough. 
+# 
+# 
+# Let's get started with the walkthrough below!
+
+# In[17]:
+
+
+import pathlib
+
+import pandas as pd
+
+# pycytominer imports
+from pycytominer.cyto_utils.cells import SingleCells
+from pycytominer import annotate, normalize, feature_select
+
+# ignore warnings
+import warnings
+
+warnings.filterwarnings("ignore")
+
+
+# ## About the inputs
+# 
+
+# In this section, we will set up the expected input and output paths that will be generated throughout this walkthrough. Let's take a look at the explanation of these inputs and outputs.
+# 
+# For this workflow, we have two main inputs:
+# 
+# - **plate_data**: This contains the quantified single-cell morphology features that we'll be working with.
+# - **plate_map**: This contains additional information related to the cells, providing valuable context of our single-cell morphology dataset.
+# 
+# Now, let's explore the outputs generated in this workflow. In this walkthrough, we will be generating four profiles:
+# 
+# - **sc_profile_path***: This refers to the single-cell morphology profile that will be generated.
+# - **anno_profile_path**: This corresponds to the annotated single-cell morphology profile.
+# - **norm_profile_path**: This represents the normalized single-cell morphology profile.
+# - **feat_profile_path**: Lastly, this refers to the selected features from the single-cell morphology profile.
+# 
+# These profiles will serve as important outputs that will help us analyze and interpret the single-cell morphology data effectively. Now that we have a clear understanding of the inputs and outputs, let's proceed further in our walkthrough.
+
+# In[18]:
+
+
+# Setting file paths
+data_dir = pathlib.Path("./data/").resolve(strict=True)
+metadata_dir = (data_dir / "metadata").resolve(strict=True)
+out_dir = pathlib.Path("results")
+out_dir.mkdir(exist_ok=True)
+
+# input file paths
+plate_data = pathlib.Path("./data/nf1_data.sqlite").resolve(strict=True)
+plate_map = (metadata_dir / "platemap_NF1_CP.csv").resolve(strict=True)
+
+# setting output paths
+sc_profile_path = out_dir / "nf1_single_cell_profile.csv.gz"
+anno_profile_path = out_dir / "nf1_annotated_profile.csv.gz"
+norm_profile_path = out_dir / "nf1_noramlzied_profile.csv.gz"
+feat_profile_path = out_dir / "nf1_features_profile.csv.gz"
+
+
+# ## Generating Merged Single-cell Morphology Dataset
+# 
+# In this section of the walkthrough, our goal is to load the NF1 dataset and create a merged single-cell morphology dataset.
+# 
+# Currently, the NF1 dataset is stored in an `sqlite` format, where each table represents a different compartment, such as Image, Cell, Nucleus, and Cytoplasm.
```

Comment:
Can you describe the origin of the sqlite file? @jenna-tomkinson can confirm, but I am fairly confident that it is a "SQLite file generated by CellProfiler exporttodatabase"

Reply:
(This is an important consideration that determines how we expect the SQLite file to be formatted)

Reply:
@axiomcura - please make sure you add this comment in the walkthrough